modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -1235,120 +1247,88 @@ protected function get_initial_config(): array {
-			elementUid = view.getIDInt().toString().substring( 0, 3 ) + tabCount,
-			tabIcon = elementor.helpers.renderIcon( view, data.tab_icon, { 'aria-hidden': true }, 'i' , 'object' );
-
-		let tabActiveIcon = tabIcon,
-			tabId = 'e-n-tab-title-' + elementUid;
+		const tabIndex = view.collection.length,
+			elementUid = view.getIDInt().toString().substr( 0, 3 ),
+			item = data,
+			hoverAnimationClass = view?.container?.settings?.attributes?.hover_animation
+				? `elementor-animation-${ view.container.settings.attributes.hover_animation }`
+				: '';
+		#>
+		<?php $this->content_template_single_item( '{{ tabIndex }}', '{{ item }}', '{{ elementUid }}', '{{ hoverAnimationClass }}' );
+	}
 
-		if ( '' !== data.tab_icon_active.value ) {
-			tabActiveIcon = elementor.helpers.renderIcon( view, data.tab_icon_active, { 'aria-hidden': true }, 'i' , 'object' );
-		}
+	protected function content_template() {
+		?>
+		<# const elementUid = view.getIDInt().toString().substr( 0, 3 ); #>
+		<div class="e-n-tabs" data-widget-number="{{ elementUid }}" aria-label="<?php echo esc_html__( 'Tabs. Open items with Enter or Space, close with Escape and navigate using the Arrow keys.', 'elementor' ); ?>">
+			<# if ( settings['tabs'] ) { #>
+			<div class="e-n-tabs-heading" role="tablist">
+				<# _.each( settings['tabs'], function( item, index ) {
+					const tabIndex = index,
+					hoverAnimationClass = settings['hover_animation']
+						? `elementor-animation-${ settings['hover_animation'] }`
+						: '';
+				#>
+				<?php $this->content_template_single_item( '{{ tabIndex }}', '{{ item }}', '{{ elementUid }}', '{{ hoverAnimationClass }}' ); ?>
+				<# } ); #>
+			</div>
+			<div class="e-n-tabs-content"></div>
+			<# } #>
+		</div>
+		<?php
+	}
 
-		const tabWrapperKey = {
-			'id': 'e-n-tab-title-' + elementUid,
-			'class': [ 'e-n-tab-title' ],
+	private function content_template_single_item( $tab_index, $item, $element_uid, $hover_animation_class ) {
+		?>
+		<#
+		const tabCount = tabIndex + 1,
+			tabId = item.element_id
+				? item.element_id
+				: 'e-n-tab-title-' + elementUid + ( tabIndex + 1 ),
+			tabUid = elementUid + tabCount,
+			tabWrapperKey = tabUid,
+			tabTitleKey = 'tab-title-' + tabUid,
+			tabIconKey = 'tab-icon-' + tabUid,
+			tabIcon = elementor.helpers.renderIcon( view, item.tab_icon, { 'aria-hidden': true }, 'i' , 'object' ),
+			activeTabIcon = item.tab_icon_active.value
+				? elementor.helpers.renderIcon( view, item.tab_icon_active, { 'aria-hidden': true }, 'i' , 'object' )
+				: tabIcon;
+
+		view.addRenderAttribute( tabWrapperKey, {
+			'id': tabId,
+			'class': [ 'e-n-tab-title',hoverAnimationClass ],
 			'data-tab-index': tabCount,
 			'role': 'tab',
 			'aria-selected': 1 === tabCount ? 'true' : 'false',
 			'tabindex': 1 === tabCount ? '0' : '-1',
-			'aria-controls': 'e-n-tab-content-' + elementUid,
+			'aria-controls': 'e-n-tab-content-' + tabUid,
 			'style': '--n-tabs-title-order: ' + tabCount + ';',
-		};
+		}, null, true );
```

Comment:
@davseve @louiswol94 
I would like to create a single template inside the editor.

I am not sure how to handle these render attributes. 

From my testing it seems safe to to apply `null, true` to both the content template and the single repeater content template, but I not 100% about this yet.

Wdyt?

Reply:
@davseve We have set the `overwrite` value to `true`.